src/pages/Home/sub-components/reason.jsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import WhatsAppIcon from "@mui/icons-material/WhatsApp";
+import LocalShippingIcon from "@mui/icons-material/LocalShipping";
+import HandshakeIcon from "@mui/icons-material/Handshake";
+import PaymentIcon from "@mui/icons-material/Payment";
```

Comment:
Code suggestion:
Can collect all of these icons in a single icons.js file and then from that icons.js file you can use it anywhere accordingly!!